plugins/example/ahda/client.py

```
@@ -0,0 +1,143 @@
+        resp.raise_for_status()
+    except requests.RequestException as e:
+        logger.error(f"Error sending webhook: {e}")
+        raise
+    return {'message': 'Webhook sent successfully'}
+
+
+@router.post('/ahda/send-webhook', tags=['ahda', 'realtime'])
+async def send_ahda_webhook(
+    uid: str = Query(...), 
+    data: dict = Body(...),
+    background_tasks: BackgroundTasks = BackgroundTasks()
```

Ruff

Do not perform function call `BackgroundTasks` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)